exercises/J-array-get-set/exercise.js

```diff
@@ -5,11 +5,13 @@
 */
 
 function first(arr) {
-  return; // complete this statement
+  return numbers[0]; { 
+    return numbers[2]
+  }// complete this statement
 }
 
 function last(arr) {
-  return; // complete this statement
+  return names[3]; // complete this statement
```

Comment:
Whilst this is technically correct, this method isn't dynamic or scalable. This function will always return the third index of an array regardless of the size of the array passed into it. A better and more dynamic way to do this would be to calculate the last index of the array passed into the function like this:

return names[names.length - 1]

This will count the amount of items in the array. The -1 will then take into account the fact array use a zero based index and calculate the correct array index for you.